lisp/__init__.py

```diff
@@ -26,12 +27,10 @@
 # Application wide "constants"
 APP_DIR = path.dirname(__file__)
 
-USER_DIR = path.join(path.expanduser("~"), ".linux_show_player")
-
-LOGS_DIR = path.join(USER_DIR, "logs")
+USER_DIRS = AppDirs("linux_show_player", __author__)
```

Comment:
I'll leave the author empty (default to application name) I think it's less confusing to have a directory with  my name around :wink:
Also, since we are already changing the setting directories, we may change it into "LinuxShowPlayer", I don't know why I've used underscores at first.
Also, I think, it could be nice to use the version parameter, only with \<major\>.\<minor\> (e.g 0.6)